onnxscript/_internal/feature_switch.py

```diff
@@ -0,0 +1,10 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+"""Switches to determine if the corresponding feature of onnxscript is enabled or not."""
+
+import os
+
+# By default: Enable
+CACHE_ORT_SESSIONS: bool = os.getenv("CACHE_ORT_SESSIONS", "1") != "0"
```

Comment:
```suggestion
cache_ort_sessions: bool = os.getenv("CACHE_ORT_SESSIONS", "1") != "0"
```

sorry - I changed my mind because it looks like it is a flag that can be changed in runtime (not constant), so snake case instead?

Reply:
> Is this in `_internal` by design? I can see arguments both ways. It should ideally be exposed to the user, but if we think this API will change, I can see a point in leaving it here.

My opinion is: Such switches should not be exposed to users calling onnx-script ops or torch_libs to construct a model. Because these switches control only some internal features of onnx-script. These internal features are not necessary to be known by these users.

Reply:
> sorry - I changed my mind because it looks like it is a flag that can be changed in runtime (not constant), so snake case instead?

I think a variable in upper case is more like a constant which should not be changed in runtime. Isn't it? And I think this is aligned with the assumption that these switches are designed to be controlled by the environment variable only.